ESpacenet.js

```diff
@@ -187,6 +181,9 @@ function scrape(doc) {
 			case "Page bookmark":
 				applyValue(newItem, label, value.firstElementChild.href)
 			break;
+			case "Application number:":
+				applyValue(newItem, label, ZU.xpathText(value,'./node()[following-sibling::a]'));
```

Comment:
There is not always such a "Global Dossier" button, see e.g. https://worldwide.espacenet.com/data/publicationDetails/biblio?locale=en_EP&II=9&FT=D&CC=AU&DB=EPODOC&NR=2814389A&date=19890601&ND=3&KC=A&adjacent=true